.github/workflows/fpga-subsystem.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     env:
       # Change this to a new random value if you suspect the cache is corrupted
       CACHE_BUSTER: 9ff0db888988
-      CALIPTRA_MCU_COMMIT: 83dbb731f10856a2b8fb488e6bdf4db0dec020d4
+      CALIPTRA_MCU_COMMIT: adb86f25ef547db46edc8ace1796013bd69c5308
```

Comment:
I think `83dbb731f10856a2b8fb488e6bdf4db0dec020d4` should be passing with a rebase. I don't want to rollback the MCU repo